{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/make_artifacts.py

```diff
@@ -39,5 +39,154 @@ def build_artifacts(location: str, output_dir: str, append: bool, verbose: int):
         How noisy the logger should be.
 
     """
-    # Stubbed out pending further consideration of best location for this code.
-    pass
+    output_dir = Path(output_dir)
+
+    if location in project_globals.LOCATIONS:
+        path = Path(output_dir) / f'{sanitize_location(location)}.hdf'
+
+        if path.exists() and not append:
+            click.confirm(f"Existing artifact found for {location}. Do you want to delete and rebuild?",
+                          abort=True)
+            logger.info(f'Deleting artifact at {str(path)}.')
+            path.unlink()
+
+        build_single_location_artifact(path, location)
+
+    elif location == 'all':
+        # FIXME: could be more careful
+        existing_artifacts = set([item.stem for item in output_dir.iterdir()
+                                  if item.is_file() and item.suffix == '.hdf'])
+        locations = set([sanitize_location(loc) for loc in project_globals.LOCATIONS])
+        existing = locations.intersection(existing_artifacts)
+
+        if existing and not append:
+            click.confirm(f'Existing artifacts found for {existing}. Do you want to delete and rebuild?',
+                          abort=True)
+            for loc in existing:
+                path = output_dir / f'{loc}.hdf'
+                logger.info(f'Deleting artifact at {str(path)}.')
+                path.unlink()
+
+        build_all_artifacts(output_dir, verbose)
```

Comment:
I think we should note in the helpstring that 'all' will trigger cluster jobs

Reply:
Done.

Reply:
Should we trigger cluster jobs without an explicit flag?  We could take `all` and build them serially in the same process.

Reply:
Test for on the cluster added.
Ticket created for serial version of "all"